.jscsrc

```diff
@@ -1,7 +1,7 @@
 {
     "disallowMixedSpacesAndTabs": true,
     "disallowTrailingWhitespace": true,
-    "validateLineBreaks": "CRLF",
+    "validateLineBreaks": "LF",
```

Comment:
Should we do this validation? AFAIK we had some promlems with line endings only in executable scripts, so CRLF vs LF should n't be a problem for plugin code.